test/BreakfastSerializer/Property/TypeHandler/DateTimeHandlerTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace BDBStudios\BreakfastSerializerTests\Property\TypeHandler;
+
+
+use BDBStudios\BreakfastSerializer\Property\TypeHandler\IsDateTime;
+use BDBStudios\BreakfastSerializerTest\Fixtures\TypeHandlers\DateTimeType;
+
+class DateTimeHandler extends \PHPUnit_Framework_TestCase
+{
+    /** @var  DateTimeType */
+    protected $instance;
+
+    const ISO_REGEX = '/^(\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2})((-|\+)\d{4})$/';
```

Comment:
`/^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(-|\+)\d{4}$/` should be enough here.
 I think the same goes for the prior.